tests/ha/check_logs.pm

```diff
@@ -8,24 +8,24 @@
 # without any warranty.
 
 # Summary: Grep logs to find segfaults
-# Maintainer: Denis Zyuzin <dzyuzin@suse.com>
+# Based on script by Denis Zyuzin
+# Maintainer: Loic Devulder <ldevulder@suse.com>
 
-use base "hacluster";
+use base 'hacluster';
 use strict;
 use testapi;
 use autotest;
 use lockapi;
 
 sub run {
     my $self = shift;
-    barrier_wait("FENCING_DONE_" . $self->cluster_name);
+    barrier_wait('FENCING_DONE_' . $self->cluster_name);
     select_console 'root-console';
 
-    script_run "hb_report -f 2014 hb_report", 120;
-    upload_logs "hb_report.tar.bz2";
-    type_string "echo segfaults=`grep -sR segfault /var/log | wc -l` > /dev/$serialdev\n";
-    die "segfault occured" unless wait_serial "segfaults=0", 60;
-    barrier_wait("LOGS_CHECKED_" . $self->cluster_name);
+    script_run q(hb_report -f 2014 hb_report), 120;
```

Comment:
The main advantage for me was the this kind of code:
  assert_script_run q(crm_mon -1 | grep 'partition with quorum')
instead of this:
  assert_script_run 'crm_mon -1 | grep \'partition with quorum\''

More readable, mainly when you have lot of quotes or double quotes :-)